docs/docs/creating-a-transformer-plugin.md

```diff
@@ -1,44 +1,41 @@
 ---
-title: Creating a Transformer Plugin
+title: Criando um Plugin de Transformação
 ---
 
-There are two types of plugins that work within Gatsby's data system, "source"
-and "transformer" plugins.
+Existem dois tipos de _plugins_ que funcionam com o sistema de dados do Gatsby, os _plugins_ de fonte de dados ("_source_") e os _plugins_ de transformação ("_transformer_").
 
-- **Source** plugins "source" data from remote or local locations into what
-  Gatsby calls [nodes](/docs/node-interface/).
-- **Transformer** plugins "transform" data provided by source plugins into new
-  nodes and/or node fields.
+- **Source** _plugins_, ou _plugins_ de fonte de dados, proveem informação de uma fonte remota ou local para o que o Gatsby chama de [nós](/docs/node-interface/).
+- **Transformer** _plugins_, ou _plugins_ de transformação, "transformam" informação provida por _source plugins_ para novos nós e/ou para campos dos nós.
 
-The purpose of this doc is to:
+O objetivo deste documento é:
 
-1.  Define what a Gatsby transformer plugin is, and
-2.  Walk through a simplified reimplementation of an existing plugin, to demonstrate how to create a transformer plugin.
+1. Definir o que é um _plugin_ de transformação do Gatbsy, e
+2. Acompanhar uma reimplementação simplificada de um _plugin_ existente para demonstrar como criar um _plugin_ de transformação.
 
-## What do transformer plugins do?
+## O que _plugins_ de transformação fazem?
 
-Transformer plugins "transform" data of one type into another type. You'll often use both source plugins and transformer plugins in your Gatsby sites.
+_Plugins_ de transformação "transformam" dados de um tipo para outro tipo. Você frequentemente usará ambos os plugins de fonte de dados e de transformação no seu website feito com Gatsby.
 
-This loose coupling between the data source and transformer plugins allow Gatsby developers to quickly assemble complex data transformation pipelines with little work.
+Esse acoplamento flexível entre os _plugins_ de fonte de dados e de transformação permitem que os desenvolvedores que usam Gatsby montem rapidamente pipelines de transformação de dados complexos com pouco trabalho.
```

Comment:
pipelines:  traduzir o termo ou pelo menos colocar em itálico

Reply:
15dd1f2